src/client/components/app/Transactions/DeployLineTx.tsx

```diff
@@ -213,25 +213,28 @@ export const DeployLineTx: FC<DeployLineProps> = (props) => {
         // displayGuidance={displaySourceGuidance}
       />
       {inputAddressWarning !== '' ? <div style={{ color: '#C3272B' }}>{inputAddressWarning}</div> : ''}
-      <TxTTLInput
+      <TxNumberInput
```

Comment:
I'm not sure why, but this change makes the deploy line modal look super weird: 

After: 
<img width="286" alt="image" src="https://user-images.githubusercontent.com/11972548/216719400-8f9cc28a-785a-4aaa-83b3-12e8c17250c7.png">

Before: 
<img width="267" alt="image" src="https://user-images.githubusercontent.com/11972548/216719533-7bd23f1f-562d-4603-a7ff-4b696f029a38.png">